langlib/lang.xml/src/main/ballerina/src/lang.xml/xml.bal

```diff
@@ -205,6 +205,16 @@ public isolated function createComment(string content) returns Comment = @java:M
     name: "createComment"
 } external;
 
+# Constructs an xml sequence representing zero of more parsed characters.
+#
+# + chars - the characters
+# + return - an xml sequence that is either empty or consists of one text item
+# The constructed sequence will be empty when the length of `chars` is zero.
```

Comment:
The above doc comment doesn't adhere to the best practices we follow. Refer to the last point in https://github.com/ballerina-platform/ballerina-distribution/blob/master/doc-guidelines.md

Reply:
I have made the changes according to the best practices, however this is the original content I took from the spec.

Reply:
@anupama-pathirage, we usually copy the content directly from the spec https://github.com/ballerina-platform/ballerina-spec/tree/master/lang/lib for langlib methods. I guess whatever documentation changes should come from the spec itself?

Reply:
Shall we revert these changes for the time-being? There are several similar usages in the langlib, I guess we can revisit them all together?

Reply:
@MaryamZi Thanks for the update. Yes. Lets stick to the spec for now and later we can consider revisting them.